gentx-validator/main.go

```diff
@@ -0,0 +1,123 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/palomachain/paloma/app"
+)
+
+var (
+	minCommissionAllowed = sdk.MustNewDecFromStr("0.02")
```

Comment:
If you don't want a min commission, you can also remove this too!

Reply:
@taariq thoughts? I think we can remove this

Reply:
we want a minimum commission so that we do not encourage race to bottom competition. I recommend we hold to 2%.